Podfile

```diff
@@ -21,7 +21,7 @@ target 'GliaWidgets' do
   pod 'lottie-ios', '3.2.3'
 end
 
-target 'GliaWidgetsTests' do
+target 'SnapshotTests' do
```

Comment:
We shouldn't lose regular tests either, right?

Reply:
Yes, regular tests still run in GliaWidgetsTests. It's even possible to use text based snapshot testing for GliaWidgetsTests, but for now I decided to avoid including it in GliaWidgetsTests to keep things simple.